test/bugsnag_test.exs

```diff
@@ -1,13 +1,14 @@
 defmodule BugsnagTest do
   use ExUnit.Case
 
+  @tag capture_log: true
```

Comment:
error output goes on screen in tests, in case when was expected, so capture_log is best for it to avoid confusion.